staging/src/k8s.io/legacy-cloud-providers/gce/gce.go

```diff
@@ -655,6 +655,11 @@ func (g *Cloud) Initialize(clientBuilder cloudprovider.ControllerClientBuilder,
 	g.eventBroadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: g.client.CoreV1().Events("")})
 	g.eventRecorder = g.eventBroadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: "g-cloudprovider"})
 
+	go func() {
```

Comment:
rather than having this as an anonymous function, it would be nice to make this an actual cleanup or shutdown method,